app/models/motion.rb

```diff
@@ -44,6 +44,8 @@ class Motion < ActiveRecord::Base
   has_paper_trail only: [:name, :description, :closing_at, :outcome]
 
   after_initialize :set_default_closing_at
+  before_validation :close_if_lapsed!
```

Comment:
I think this is totally okay to go in, but want to keep my ear out for a better way. Here's what makes me a little uncomfortable:
- I think it's a little odd to have such a slow event (load up a bunch of relations, create a bunch of models, then save = lots of stuff for the user to wait on) happen before validation.
- 'before validation' happens quite a lot, esp. in the test suite. Also, we'd get slightly weird behaviour if a motion is invalid, tries to save, but still closes. ('I'm updating the title to blank; whoops that didn't work. Wait why is it closed now, and I can't edit anymore?')

I also wonder if it'd be possible to bump the DidNotVotes creation (which is the slow bit) off to a side job after the motion has closed?

```
EventBus.listen('motion_closed', 'motion_closed_by_user') do |motion|
  motion.did_not_votes.delete_all
  motion.did_not_votes.create!((motion.group_members - motion.voters).map { |user| {motion: motion, user: user} }
end
```

💛